DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/loop_carried_dependency/README.md

```diff
@@ -173,14 +173,15 @@ After learning how to use the extensions for Intel oneAPI Toolkits, return to th
      ```
 
 *Note:* The Intel® PAC with Intel Arria® 10 GX FPGA and Intel® FPGA PAC D5005 (with Intel Stratix® 10 SX) do not support Windows*. Compiling to FPGA hardware on Windows* requires a third-party or custom Board Support Package (BSP) with Windows* support.
+*Note:* If you encounter any issues with long path lengths when compiling under Windows*, you may have to create your ‘build’ directory in a shorter path, for example c:\samples\build.  You can then run cmake from that directory, and provide cmake with the full path to your sample directory.
```

Comment:
I know I wrote this, but please change 'issues with long path lengths' to 'issues with long paths'